src/widgets/widget_definitions.rs

```diff
@@ -602,6 +600,15 @@ fn parse_align(o: &str) -> Result<gtk::Align> {
     })
 }
 
+/// @var duration - positive integer
+fn parse_duration(d: i32) -> Result<u32> {
+    Ok( if d.is_positive() {
+        d as u32
+    } else {
+        bail!(r#"Couldn't parse duration: '{}'. The duration must be a positive integer"#, d)
+    } )
+}
+
```

Comment:
bruh i was talking about this, lol https://github.com/elkowar/eww/blob/61e42c9c8acb53dbd2eb83ae1f5a946dabede75f/src/util.rs#L100

Reply:
Kekw